gammapy/image/profile.py

```diff
@@ -194,6 +196,241 @@ def save(self, filename):
         raise NotImplementedError
 
 
+class ImageProfile(object):
+    """
+    Image profile class.
+
+    The image profile data is stored in `~astropy.table.Table` object, with the
+    following columns:
+
+        * `x_ref` Coordinate bin center (required). 
+        * `x_min` Coordinate bin minimum (optional).
+        * `x_max` Coordinate bin maximum (optional).
+        * `profile` Image profile data (required).
+        * `profile_err` Image profile data error (optional).
+
+    """
+    def __init__(self, table):
+        self.table = table
+
+    def smooth(self, kernel='box', radius=0.1 * u.deg, **kwargs):
+        """
+        Smooth profile with error propagation.
+
+        Smoothing is described by a convolution:
+
+        .. math::
+
+                x_j = \sum_i x_{(j - i)} h_i
+
+        Where :math:h_i are coefficients of the convolution kernel.
+
+        Parameters
+        ----------
+        kernel : {'gauss', 'box'}
+            Kernel shape
+        radius : `~astropy.units.Quantity` or float
+            Smoothing width given as quantity or float. If a float is given it
+            interpreted as smoothing width in pixels. If an (angular) quantity
+            is given it converted to pixels.
```

Comment:
> If a float is given it interpreted as smoothing width in pixels.

I don't think that's true, you always execute:
```
radius = np.abs(radius / np.diff(self.x_ref))[0]
width = 2 * radius.value + 1
```

I think I would prefer to only support float width in pix here, because smoothing is done in pix.
If you keep the quantity width, you should document that `width = self.x_ref[1] - self.x_ref[0]` is used to convert to float.

Looking at http://docs.gammapy.org/en/latest/api/gammapy.image.SkyImage.html#gammapy.image.SkyImage.smooth I see that the links to `uniform_filter` and `gaussian_filter` in scipy aren't working.
Probably it has to be `~scipy.ndimage.uniform_filter` and `~scipy.ndimage.gaussian_filter`?
Can you please fix this there and here?